cdap-docs/admin-manual/source/installation/installation.rst

```diff
@@ -98,6 +98,23 @@ in addition to having CPUs with a minimum speed of 2 GHz:
 | **CDAP Authentication Server**        | RAM                | 1 GB minimum, 2 GB recommended                |
 +---------------------------------------+--------------------+-----------------------------------------------+
 
+.. _install-hardware-memory-core-requirements:
+
+Memory and Core Requirements
+............................
+Memory and core requirements are governed by two sources: CDAP and YARN. The default
+settings for CDAP are found in the :ref:`cdap-defaults.xml <appendix-cdap-default.xml>`,
+and are overridden in particular instances by the :ref:`cdap-site.xml
+<appendix-cdap-site.xml>` file. These vary with each service and range from 512 to 1024 MB
+and from one to two cores.
+
+The YARN settings will override these; for instance, the minimum YARN container size is
+determined by ``yarn.scheduler.minimum-allocation-mb``. The YARN default in Hadoop is 1024
+MB, so containers will be allocated with 1024 MB, even if the CDAP settings are for 512 MB.
+
+With the default YARN settings, CDAP can require from 14 to 16 GB of memory, available to
+YARN, just to start.
```

Comment:
`With the default YARN and CDAP settings,`

`of memory and 14 to 16 CPU cores, available to YARN`